docs/_integration-with-exisiting-apps-java.md

```diff
@@ -57,14 +57,27 @@ Add `node_modules/` to your `.gitignore` file.
 
 ### Configuring maven
 
-Add the React Native and JSC dependency to your app's `build.gradle` file:
+Add the React Native and JavaScript engine dependencies to your `app/build.gradle` file:
 
 ```gradle
```

Comment:
This script is not correct, given the newest version of React Native 0.71.0.
I'd rather document the new template